event_registration_cancel_reason/tests/__init__.py

```diff
@@ -0,0 +1,3 @@
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
```

Comment:
No need of license lines on __init__

Reply:
Well, it's common practice. What we remove is copyright in these files.